docs/docs-beta/docs/tutorials/category-two/prompt-engineering/additional-prompt.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Additional Prompt
+description: Using an additional prompt
+last_update:
+  author: Dennis Hume
+sidebar_position: 50
+---
+
+For our final prompt engineering asset, we can reuse many of the concepts we have already learned. We can start with the prompt itself. Unlike the original prompt which would come in from the user, we can be very specific with the input value for the prompt. In this case we will provide the current time (we can ignore timezones for this example) and the hours of operation from the fuel station. As with the last prompt, we will also ensure the output is in JSON. To accomplish this a prompt might look like:
+
+<CodeExample path="project_prompt_eng/project_prompt_eng/assets.py" language="python" lineStart="34" lineEnd="66"/>
+
+Now that we have the prompt we can create our asset. The asset will combine the current time with the alternative fuel stations data. It will then generate a prompt for each fuel station to ask the AI model and determine if the fuel station is currently available.
```

Comment:
```suggestion
Now that we have the prompt, we can create our asset. The asset will combine the current time with the alternative fuel stations data. It will then generate a prompt for each fuel station to ask the AI model and determine if the fuel station is currently available.
```